packages/flutter/test/material/control_list_tile_test.dart

```diff
@@ -34,20 +34,232 @@ void main() {
     expect(log, equals(<dynamic>[false, '-', false]));
   });
 
-  testWidgets('RadioListTile control test', (WidgetTester tester) async {
+  testWidgets('RadioListTile should initialize according to groupValue', (WidgetTester tester) async {
+    final List<int> values = <int>[0, 1, 2];
+    int selectedValue;
+    // Input parameters are required for [RadioListTile], but they are
+    // irrelevant when searching with [find.byType].
+    final Type radioListTileType = const RadioListTile<int>(
+      value: 0,
+      groupValue: 0,
+      onChanged: null,
+    ).runtimeType;
+    List<RadioListTile<int>> generatedRadioListTiles;
+
+    await tester.pumpWidget(wrap(
+      child: StatefulBuilder(
+        builder: (BuildContext context, StateSetter setState) {
+          return Scaffold(
+            body: ListView.builder(
+              itemCount: values.length,
+              itemBuilder: (BuildContext context, int index) => RadioListTile<int>(
+                onChanged: (int value) {
+                  setState(() { selectedValue = value; });
+                },
+                value: values[index],
+                groupValue: selectedValue,
+                title: Text(values[index].toString()),
+              ),
+            ),
+          );
+        },
+      ),
+    ));
+
+    generatedRadioListTiles = find
```

Comment:
This is pretty slick!

You could define a local function here, since it's used twice